geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/accumulo/tools/AccumuloRunner.scala

```diff
@@ -101,48 +98,6 @@ trait RunnerWithAccumuloEnvironment extends Runner {
           }
         }
       }
-
-      if (p.instance == null) {
```

Comment:
I'm fine with this code going.  Do we want any of its logic?

Reply:
the underlying accumulo classes don't exist anymore, so i'm not sure we can do the same thing. Accumulo 2 uses a `accumulo-client.properties` file that you can specify connection params in - we should probably open a ticket to support that, but it seemed a little more complicated than i wanted to do at the moment (we'd have to make all the ds params optional for one thing).

Reply:
opened https://geomesa.atlassian.net/browse/GEOMESA-2800

Reply:
Yeah, that's great separate ticket.  Thanks for opening it.